### PR TITLE
Edits to sign-in page

### PR DESCRIPTION
**Pull request by achou11**
A couple of small changes:

- Autofocuses the password input on initial render. Super small but was bugging me a little 😅 
- Applies focus state styling identical to the hover state for the submit button (evident when tab navigating)

### CODE REVIEW

File: src/pages/SignIn.js [hunk 1]
```diff
@@ -89,6 +89,7 @@ const SignIn = ({ network, initNetwork, eraseNetwork }) => {
         className='form-input'
         type='password'
         required
+        autoFocus
```
Comment (achou11) [inline src/pages/SignIn.js]: Amazingly, I always thought that this was supposed to be `autofocus` in React, but apparently it's camel-cased??

File: src/index.css [hunk 2]
```diff
@@ -92,7 +92,7 @@ hr {
   color: #afafaf;
 }
 
-.button:hover:not(:disabled) {
+.button:hover:not(:disabled), .button:focus:not(:disabled) {
```
Comment (achou11) [inline src/index.css]: Guess this would apply to all buttons with `.button` class. Not sure if this would be a problem for other buttons in the app.

Comment (achou11) [reply src/index.css]: Taking a closer look and thinking that maybe I can relocate this to be more less global. Let me know what you think!

**Comment by indutny** [comment]
Thanks!

**Comment by achou11** [comment]
> LGTM. Should I land it now?

Sounds good!

**Comment by indutny** [comment]
Landed in 7dc599ca1309dd12e907f6eca944b3b59c4c5f68 and 6d0c245489395482d6de6c475c4fbfb16e8303a3. Thank you!

**Pull request closed without merging**